ayaka_v8.js

```diff
@@ -169,167 +165,166 @@ function 韻母規則() {
       ]],
     ]],
 
-    // 遇攝
-    ['魚韻', [
-      ['莊組', 'o'],
-      ['舌齒音', 'yo'],
-      ['牙喉音', 'yo'],
-    ]],
-    ['虞韻', [
-      ['脣音', 'u'],
-      ['來母', 'u'],
-      ['知組', 'yuu'],
-      ['莊組', 'u'],
-      ['舌齒音', 'yu'],
-      ['以母', 'yu'],
-      ['牙喉音', 'u'],
-    ]],
-    ['模韻', [
-      ['脣音', 'o'],
-      ['舌齒音', 'o'],
-      ['影母', 'wo'],
-      ['牙喉音', 'o'],
+    ['遇攝', [
+      ['魚韻', [
+        ['莊組', 'o'],
+        ['舌齒音', 'yo'],
+        ['牙喉音', 'yo'],
```

Comment:
```suggestion
        ['莊組', 'o'],
        ['', 'yo'],
```
類似這種是不是都應該改成這樣？將「不發生音變」的環境逐個寫出是多此一舉，而且從原理上來說它是不能枚舉的

Reply:
> 而且從原理上來說它是不能枚舉的

對欸w

想起來這其實是曾經這些個方案依靠的「表格整理」的遺留【而且最近提到的那個 extrapolation-tool 就是輔助做整理表用的😂